hbase-server/src/test/java/org/apache/hadoop/hbase/mob/TestMobFileName.java

```diff
@@ -47,6 +47,7 @@
   private Date date;
   private String dateStr;
   private byte[] startKey;
+  private String regionName = "region";
```

Comment:
Should also have tests for how `hashcode` and `equals` behave in the case where we get a MobFileName that doesn't include a region (e.g. when there are mob hfiles from prior to this patch)

Reply:
MobFileName code is not used during upgrade from older version and does not have any code which handles old files.